gilda/tests/test_process.py

```diff
@@ -24,3 +25,8 @@ def test_roman_arabic():
     assert replace_roman_arabic('xx-I') == 'xx-1'
     assert replace_roman_arabic('xx viii') == 'xx 8'
     assert replace_roman_arabic('xx-iX') == 'xx-9'
+
+
+def test_normalize():
+    assert normalize('Löfgren’s syndrome') == 'lofgren\'s syndrome'
```

Comment:
Is there an expansion somewhere for Löfgren to Loefgren?

Maybe this is tricky if it's German specific. I wonder if the umlaut rule applies to other languages that use this diacritic. @kkaris Swedish has lots of these! Maybe you can tell us more :)